css/README.md

```diff
@@ -1,364 +1,350 @@
-  // ...
-}
-
-.ddl-item {
-  // ...
+[aria-hidden] {
+    /* ... */
 }
 ```
 
-##### Bad
+### Properties
+
+Finally, properties are what give the selected elements of a rule declaration their style. Properties are key-value pairs, and a rule declaration can contain one or more property declarations. Property declarations look like this:
 
 ```css
-.item-list {
-  // ...
+/* some selector */ {
+    background: #f1f1f1;
+    color: #333;
 }
+```
+
+## CSS
 
-.dropdown-item-list {
-  // ...
+Nothing fancy since it's compiled by the Sass compiler.
+
+## Sass
+
+As written in the introduction we use the [RSCSS](http://rscss.io/) system. That means we only have components besides generic styles. A component can be a collection of components compositing their collective appearance, but nothing more or less.
+
+### Format
+
+- Use soft tabs with **4** (four) spaces, they're the only way to guarantee code renders the same in any environment.
+- When grouping selectors, keep individual selectors to a single line.
+- Include one space before the opening brace of declaration blocks for legibility.
+- Place closing braces of declaration blocks on a new line.
+- Use double quotes consistently.
+- Include one space after `:` for each declaration.
+- Each declaration should appear on its own line for more accurate error reporting.
+- End all declarations with a semi-colon. The last declaration's is optional, but your code is more error prone without it.
+- Comma-separated property values should include a space after each comma (e.g., `box-shadow`).
+- Include spaces after commas *within* `rgb()`, `rgba()`, `hsl()`, `hsla()`, or `rect()` values.
+- Prefix property values or color parameters with a leading zero (e.g., `0.5` instead of `.5` and `-0.5px` instead of `-.5px`).
+- Lowercase all hex values, e.g., `#fff`. Lowercase letters are much easier to discern when scanning a document as they tend to have more unique shapes.
+- Use shorthand hex values where available, e.g., `#fff` instead of `#ffffff`.
+- Quote attribute values in selectors, e.g., `input[type="text"]`. [They’re only optional in some cases](http://mathiasbynens.be/notes/unquoted-attribute-values#css), and it’s a good practice for consistency.
+- Avoid specifying units for zero values, e.g., `margin: 0;` instead of `margin: 0px;`.
+- Use the `.scss` syntax, never the original `.sass` syntax.
+
+
+```scss
+// ✗ Bad SCSS
+.selector, .selector-secondary, .selector[type=text] {
+  padding:15px;
+  margin:0px 0px 15px;
+  background-color:rgba(0,0,0,0.5);
+  box-shadow:0px 1px 2px #CCC,inset 0 1px 0 #FFFFFF
 }
 
-.xyz-item-list {
-  // ...
+// ✓ Better SCSS
+.selector,
+.selector-secondary,
+.selector[type="text"] {
+    background-color: rgba(0, 0, 0, 0.5);
+    box-shadow: 0 1px 2px #ccc, inset 0 1px 0 #fff;
+    margin-bottom: 15px;
+    padding: 15px;
 }
 ```
 
-<a name="comments"></a>
-## Comments
+### Ordering of property declarations
 
-* Keep line-length to a sensible maximum, e.g., 80 columns.
```

Comment:
@mzdr Can we agree on a bigger value here? 80 is very old and was mainly introduced because of the small screens. However, it is quite little on today's screens. How about 120?